interactions/ban/ban.go

```diff
@@ -33,11 +29,147 @@ var BanCommand = discord.SlashCommandCreate{
 	DMPermission:             utils.Ref(false),
 	DefaultMemberPermissions: json.NewNullablePtr(discord.PermissionBanMembers),
 	Options: []discord.ApplicationCommandOption{
-		banWithMessageSubCommand,
-		banUntilSubcommand,
+
+		discord.ApplicationCommandOptionUser{
+			Name:        "user",
+			Description: "The user to ban",
+			Required:    true,
+		},
+		discord.ApplicationCommandOptionString{
+			Name:        "reason",
+			Description: "Reason for banning the user. Not sent to the user.",
```

Comment:
Should update the description here to let people know if not provided it defaults to `message`.